ref/Microsoft.Build/net/Microsoft.Build.cs

```diff
@@ -1413,11 +1413,12 @@ public sealed partial class ProjectGraph
         public ProjectGraph(string entryProjectFile, Microsoft.Build.Evaluation.ProjectCollection projectCollection, System.Collections.Generic.Dictionary<string, string> globalProperties, string toolsVersion) { }
         public Microsoft.Build.Graph.ProjectGraphNode EntryProjectNode { get { throw null; } }
         public System.Collections.Generic.IReadOnlyCollection<Microsoft.Build.Graph.ProjectGraphNode> ProjectNodes { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyDictionary<Microsoft.Build.Graph.ProjectGraphNode, string[]> GetTargetLists(string[] entryProjectTargets) { throw null; }
```

Comment:
It needed to be public since it's called from XMake.cs, which is in another assembly.

Depending on how the scheduling bits take shape though, we can potentially remove this once we stop treating the scheduler as a black-box.